ra2ce/analysis/indirect/multi_link_losses.py

```diff
@@ -8,37 +8,33 @@
     AnalysisSectionIndirect,
 )
 from ra2ce.analysis.analysis_config_data.enums.loss_type_enum import LossTypeEnum
+from ra2ce.analysis.analysis_input_wrapper import AnalysisInputWrapper
 from ra2ce.analysis.indirect.analysis_indirect_protocol import AnalysisIndirectProtocol
 from ra2ce.analysis.indirect.multi_link_redundancy import MultiLinkRedundancy
 from ra2ce.network.graph_files.graph_file import GraphFile
 from ra2ce.network.hazard.hazard_names import HazardNames
 
 
 class MultiLinkLosses(AnalysisIndirectProtocol):
-    graph_file: GraphFile
+    analysis_input: AnalysisInputWrapper
```

Comment:
I do not think you should be keeping this property here. It should be consumed during the `__init__` therefore its contents "unwrapped" into explicit properties for the class.

Reply:
Made private.